server/database/tests/index.test.js

```diff
@@ -1,7 +1,8 @@
 import SequelizeMock from 'sequelize-mock';
 import { resetAndMockDB } from '@utils/testUtils';
 import { DB_ENV } from '@utils/testUtils/mockData';
-import * as pg from 'pg';
+const pg = require('pg');
```

Comment:
lets use import instead of require

Reply:
If we use import instead of require, the test case will fail, because in the dbConfig file we are using require for pg.

Reply:
I don't follow this one. Everything in this file is an import. If the test is failing we must fix that one

Reply:
Okey.. resolved it.